### DISCUSSION TITLE
Flutter SDK http package dependency

### DISCUSSION THREAD

**Discussion by EricNahon** [pocketbase/pocketbase#2986] [Q&A]
Hi,

First of all, this pocketbase project is awsome, congrats!
Would it be possible to upgrade the http dependency of the Flutter SDK from http ^0.13.4 to http ^1.1.0 ?

Best,
Eric

**Comment by ganigeorgiev** [comment]
The Dart SDK will be updated but that will be after PocketBase v0.17.0 release.

You can also check https://github.com/pocketbase/dart-sdk/issues/42.

**Reply by EricNahon** [reply]
Great news!

Thanks for your ultra-fast response :)
